javascript/javascript1/week1/js-week1.js

```diff
@@ -0,0 +1,82 @@
+// Smart-ease
+// Age-ify (A future age calculator)
+const yearOfBirth = 1975;
+const yearFuture = 2045;
+
+let age = yearFuture - yearOfBirth;
+// console.log(age);
+
+console.log("You will be " + age + " " + "years old in " + yearFuture + ".");
+
+
+
+// Goodboy-Oldboy (A dog age calculator)
+const dogYearOfBirth = 2020;
+const dogYearFuture = 2035;
+let dogYear = dogYearFuture - dogYearOfBirth;
+let shouldShowResultInDogYears = true;
+
+if (shouldShowResultInDogYears) {
+    dogYear = dogYear * 7;
+    console.log("Your dog will be " + dogYear + " " + "dog years old in " + dogYearFuture + ".");
+}
+else {
+    console.log("Your dog will be " + dogYear + " " + "human years old in " + dogYearFuture + ".")
+}
+
+
+
+// House Pricey (A house price calculator)
+const peterHouseWidth = 8;
```

Comment:
Imagine you can put all values in the same array for both Peter and Julia, for example,
`const volumeInMeters = [8*10*10,5*11*8]` `const gardenSizeInM2 = [100,70]`
where you know that volumeInMeters[0] is Peter's house volume, and another one is Julia's etc.
Can you refactor the rest of your code so you will have fewer lines of code?